src/main/webapp/academicAdminOffice/student/registration/viewRegistrationDetails.jsp

```diff
@@ -216,7 +217,21 @@
 		</academic:allowed>
 	</logic:notEmpty>
 	
-	
+	<%-- Registration Data by Execution Year --%>
+	<h3 class="mbottom05 mtop25 separator2"><bean:message key="title.registrationDataByExecutionYear" bundle="ACADEMIC_OFFICE_RESOURCES"/></h3>
+	<fr:view name="registration" property="registrationDataByExecutionYear">
```

Comment:
Please add an indication for the Empty State, as per https://confluence.fenixedu.org/display/FENIXEDU/Empty+state.